src/test/java/org/fundacionjala/coding/abel/SortingOnPlanetTwisted37Test.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.abel;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+/**
+ *
+ */
+public class SortingOnPlanetTwisted37Test {
+
+    private static final Integer[] EXPECTATIVE_1 = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private static final Integer[] EXPECTATIVE_2 = new Integer[]{12, 14, 13};
+    private static final Integer[] EXPECTATIVE_3 = new Integer[]{2, 7, 4, 3, 9};
+    private static final Integer[] INPUT_1 = new Integer[]{1, 2, 3, 4, 5, 6, 7, 8, 9};
+    private static final Integer[] INPUT_2 = new Integer[]{12, 13, 14};
+    private static final Integer[] INPUT_3 = new Integer[]{9, 2, 4, 7, 3};
+
+    /**
+     *
+     */
+    @Test
+    public void basicTests1() {
+        SortingOnPlanetTwisted37 c = new SortingOnPlanetTwisted37();
```

Comment:
You are repeating this line of code in all your unit tests.
Please move it to an @Before method.